platforms/ios/framework/src/TGMapView.mm

```diff
@@ -231,6 +233,7 @@ - (void)setupGestureRecognizers
     _rotationGestureRecognizer = [[UIRotationGestureRecognizer alloc] initWithTarget:self action:@selector(respondToRotationGesture:)];
     _shoveGestureRecognizer = [[UIPanGestureRecognizer alloc] initWithTarget:self action:@selector(respondToShoveGesture:)];
     _shoveGestureRecognizer.minimumNumberOfTouches = 2;
+    _shoveGestureRecognizer.maximumNumberOfTouches = 2;
```

Comment:
fyi google maps allows 2+ touches for shoving. Yes client app can override this but maybe default should allow 2+ touches for shoving. Thoughts?

Reply:
Apple Maps only allows 2, so I don't think this will be unexpected to iOS users. We already enforced this anyway, just in the gesture responder rather than in the recognizer itself.

Reply:
Cool, I did notice the check of "touches != 2" there. We have not touched any of gesture handling code in ages :D.